tester_app/client/assets/fonts

```diff
@@ -0,0 +1 @@
+/Users/Rob/dev/open-source/react_on_rails-generator-results/tester_app/app/assets/fonts
```

Comment:
Yikes! this is super bad.
We need to be sure the generator uses relative paths!